macos/Classes/FlutterRTCMediaStream.m

```diff
@@ -290,6 +290,9 @@ - (void)getUserVideo:(NSDictionary *)constraints
     
     if (videoDevice) {
         RTCVideoSource *videoSource = [self.peerConnectionFactory videoSource];
+        if(self.videoCapturer){
```

Comment:
Hm, I see inconsistent with previous code formating here ☹️ - whitespace before and after the condition is absent. But, unfortunately, PR already merged. @cloudwebrtc could I add formatting fixing commit directly to master?

Reply:
@ycherniavskyi No problem, you can fix it directly.